READMEs/search-and-filter-assets.md

```diff
@@ -0,0 +1,104 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+
+# Quickstart: Search Assets Flow
+
+This quickstart describes how assets can be found by their `tags` from Aquarius.
+
+
+Here are the steps:
+
+1.  Setup
+2.  Alice creates few assets for testing
+3.  Alice searches & filters assets by their `tags`
+
+Let's go through each step.
+
+## 1. Setup
+
+From [data-nfts-and-datatokens-flow](data-nfts-and-datatokens-flow.md), do:
+- [x] Setup : Prerequisites
+- [x] Setup : Download barge and run services
+- [x] Setup : Install the library
+- [x] Setup : Set envvars
+- [x] Setup : Setup in Python
+
+
+## 2. Alice publishes datasets
+
+Now, you're Alice. Using [publish-flow](publish-flow.md) model, do:
+
+```python
```

Comment:
We need to avoid having 50 lines of boilerplate. There's been great effort to fix this in ocean.py in the previous 8 weeks. I am loathe to reintroduce it.

And in general this is also a big DRY violation. We have to be working to reduce DRY violations, not the other way around.)

Solution: use "create_url_asset()" which is one line.

Then edit the ddo's tags via the README. Should be just 1-3 lines too.

That's it.

Reply:
I need to update the metadata which contains `tags` key, that's why I expanded the functionality, should I update `_create1` function in order to receive `metadata` as parameter?

Reply:
> should I update _create1 function in order to receive metadata as parameter

No. Edit it after. That's why: "**edit** the ddo's tags". 

It's useful to show the user how to do this too.